helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -97,10 +98,25 @@ public ParticipantManager(HelixManager manager, HelixZkClient zkclient, int sess
   }
 
   /**
-   * Handle new session for a participang.
-   * @throws Exception
+   * Handles a new session for a participant. The new session's id is passed in when participant
+   * manager is created, as it is required in
+   * {@link org.apache.helix.manager.zk.zookeeper.ZkClient#createEphemeral(String, Object, String)}
```

Comment:
The explanation is good here, but this link to the code is not necessary. And maybe hard for us to maintain.

Reply:
Sure. Will address this link.